src/eddymotion/model/base.py

```diff
@@ -28,13 +28,21 @@
 from dipy.core.gradients import gradient_table
 from joblib import Parallel, delayed
 
+a_min_S0 = 1e-5
+a_max_S0 = 1.0
+bval_upper_cap = 1000
+bval_lower_th = 50
+bval_upper_th = 10000
+percentil_percentage = 75
+timeframe_midpoint_tol = 1e-2  # in seconds
+
 
 def _exec_fit(model, data, chunk=None):
     retval = model.fit(data)
     return retval, chunk
 
 
-def _exec_predict(model, gradient, chunk=None, **kwargs):
+def _exec_predict_dwi(model, gradient, chunk=None, **kwargs):
```

Comment:
I renamed this method to contain the `dwi` label, as it requires a gradient and optionally uses a `S0` argument. If `gradient` in reality should be an `index`, it may be renamed back. Also, as things are right now, I do not see the need to pass an `S0` either.